Cubical/Structures/Monoid.agda

```diff
@@ -43,7 +41,7 @@ record IsMonoid {A : Type ℓ} (ε : A) (_·_ : A → A → A) : Type ℓ where
 
 
 record Monoid : Type (ℓ-suc ℓ) where
-
+  no-eta-equality
```

Comment:
Same here. I'm surprised you need this

Reply:
Just checked. Don't think I do need it in the end. Do you want me to remove it? I've grown quite fond of no-eta after working on this project...

Reply:
Yeah, please remove it. I think we should have as much eta as possible as it simplifies proofs.